packages/react-admin-stories/src/react-admin-rte/StyleContent.tsx

```diff
@@ -0,0 +1,36 @@
+import { storiesOf } from "@storybook/react";
+import { IRteRef, makeRteApi, Rte } from "@vivid-planet/react-admin-rte";
+import * as React from "react";
+import { exampleContent, PrintEditorState, RteLayout, useAutoFocus } from "./helper";
+import styled from "styled-components";
+
+const [useRteApi] = makeRteApi();
+
+const StyledRte = styled(Rte)`
+    div.DraftEditor-root {
+        font-family: monospace;
```

Comment:
Sollte das Font-Styling im RTE nicht vom ausgewählten Text-Stil abhängig sein?  
Außerdem sind Mono-Fonts generell eigentlich nicht so schön und eine `line-height` von `2` ist auch etwas viel.

Reply:
> Sollte das Font-Styling im RTE nicht vom ausgewählten Text-Stil abhängig sein?
> Außerdem sind Mono-Fonts generell eigentlich nicht so schön und eine `line-height` von `2` ist auch etwas viel.

Generelle stylings wie font-family oder line height, will man in der Regel für den gesamten Content in der Editor-Darstellung einheitlich haben.
Font-size ist natürlich vom Text-Stil abhängig. 
Monospace und line height 2 sind nur Beispielwerte.

Reply:
Es kommt aber sicher öfters vor, dass es Text-Stile mit verschiedenen fonts, line-height's gibt.  
So würde man ja den unterschied garnicht sehen.